File: SOKOBAN/src/code/objetmobile.java
package code;

public class objetmobile extends objet{
	
	String img;

	public objetmobile(int x, int y) {
		super(x, y);
	}
	void deplacerDe(int dx,int dy){
	    setX(getX() + dx);
	    setY(getY() + dy);
	}
}

File: MonSokoban/objetmobile.cpp
#include "objetmobile.h"

ObjetMobile::ObjetMobile(int x, int y):Objet(x,y){
}

void ObjetMobile::deplacerDe(int dx,int dy){
    this->setX(this->getX() + dx);
    this->setY(this->getY() + dy);
}
